test/powershell/engine/Cdxml/Cdxml.Tests.ps1

```diff
@@ -0,0 +1,268 @@
+$script:CimClassName = "PSCore_CimTest1"
+$script:CimNamespace = "root/default"
+$script:moduleDir = join-path $PSScriptRoot assets CimTest
+$script:deleteMof = join-path $moduleDir DeleteCimTest.mof
+$script:createMof = join-path $moduleDir CreateCimTest.mof
+
+$CimCmdletArgs = @{
+    Namespace = ${script:CimNamespace}
+    ClassName = ${script:CimClassName}
+    ErrorAction = "SilentlyContinue"
+    }
+
+$script:ItSkipOrPending = @{}
+
+function Test-CimTestClass {
+    (Get-CimClass @CimCmdletArgs) -ne $null
+}
+
+function Test-CimTestInstance {
+    (Get-CimInstance @CimCmdletArgs) -ne $null
+}
+
+Describe "Cdxml cmdlets are supported" -Tag CI,RequireAdminOnWindows {
+    BeforeAll {
+        $skipNotWindows = ! $IsWindows
+        if ( $skipNotWindows ) {

```

Comment:
I thought it was clearer here `-skip` (I've seen other tests do exactly the opposite of what was wanted)